### DISCUSSION TITLE
Tutorial error

### DISCUSSION THREAD

**Discussion by prospersu** [Xilinx/finn#1208] [Q&A]
Dear Team Finn, 

I tired to follow the FINN Example FPGA Flow Using MNIST Numerals example,, and I can compile the codes successfully. However, the results are wrong as shown below. I tried to debug myself to see if the test bench is wrong or not, but I still can't find it. Please guide me where I should fix it to get the right results.
![Screenshot 2024-10-08 153642](https://github.com/user-attachments/assets/f39b1c63-ad9f-48f6-896b-b37b5a059ba3)
![Screenshot 2024-10-08 153704](https://github.com/user-attachments/assets/81f5fd69-0df1-42fb-8583-fa4845adf322)

Thank you so much for your time and support. 

Sincerely regards,
prospersu

**Comment by auphelia** [comment]
Hi @prospersu ,

Thanks for bringing that up, could you turn your discussion into a bug report in issues: https://github.com/Xilinx/finn/issues
We will have a look at it as soon as possible.

**Reply by prospersu** [reply]
Dear auphelia,

Thank you so much for your fast reply. I have created the post for this bug just now, looking forward to hear from your team soon. Thanks again.

Regards,
prospersu
